installer_scripts/install-yb-voyager

```diff
@@ -905,6 +909,27 @@ create_guardrail_scripts_dir() {
 	fi
 }
 
+create_config_templates_dir() {
+	scripts_parent_dir="/opt/yb-voyager"
+	scripts_dir_name="config-templates"
+	sudo mkdir -p $scripts_parent_dir/$scripts_dir_name
+
+	output "Installing the config templates"
+	if [ "${VERSION}" != "local" ]
+	then
+		TAR_URL="https://github.com/yugabyte/yb-voyager/archive/$YB_VOYAGER_GIT_HASH.tar.gz"
+		sudo wget -nv -O /tmp/yb-voyager.tar.gz $TAR_URL
```

Comment:
we seem to be downloading the entire tar multiple times now. (one for guardrails scripts, one for config templates). This isn't ideal. 

Can we either:
- download it once, and extract whatever we want? 
- only download the specific folder that we're interested in? ( Idk if that's possible )

Reply:
For now I am keeping it as it is. It is taking some time to properly add and test the flow. Adding a ticket for this:
https://yugabyte.atlassian.net/browse/DB-16693